`PathsConfig` is not registered in the state operator

State validation fails on the state generated by the current `main`:

```

java.lang.RuntimeException: java.lang.IllegalArgumentException: No config data record available of type 'class org.hiero.consensus.config.PathsConfig'
	at com.hedera.statevalidation.util.StateUtils.initState(StateUtils.java:205)
	at com.hedera.statevalidation.util.StateUtils.getState(StateUtils.java:129)
	at com.hedera.statevalidation.util.StateUtils.getDefaultState(StateUtils.java:119)
	at com.hedera.statevalidation.ValidateCommand.call(ValidateCommand.java:119)
	at com.hedera.statevalidation.ValidateCommand.call(ValidateCommand.java:37)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2031)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
	at picocli.CommandLine.execute(CommandLine.java:2174)
	at com.hedera.statevalidation.StateOperatorCommand.main(StateOperatorCommand.java:48)
Caused by: java.lang.IllegalArgumentException: No config data record available of type 'class org.hiero.consensus.config.PathsConfig'
	at com.swirlds.config.impl.internal.ConfigDataService.getConfigData(ConfigDataService.java:96)
	at com.swirlds.config.impl.internal.ConfigurationImpl.getConfigData(ConfigurationImpl.java:233)
	at com.hedera.statevalidation.util.PlatformContextHelper$1.getFileSystemManager(PlatformContextHelper.java:56)
	at com.hedera.statevalidation.util.StateUtils.initState(StateUtils.java:187)
	... 13 more
```

Likely, this failure is related to #25292

--- a/hedera-state-validator/src/main/java/com/hedera/statevalidation/util/ConfigUtils.java
+++ b/hedera-state-validator/src/main/java/com/hedera/statevalidation/util/ConfigUtils.java
@@ -49,6 +49,7 @@ import com.swirlds.platform.builder.ModulesConfig;
 import com.swirlds.virtualmap.config.VirtualMapConfig;
 import org.hiero.base.crypto.config.CryptoConfig;
 import org.hiero.consensus.config.BasicConfig;
+import org.hiero.consensus.config.PathsConfig;
 import org.hiero.consensus.metrics.config.MetricsConfig;
 import org.hiero.consensus.pces.config.PcesConfig;
 import org.hiero.consensus.state.config.StateConfig;
@@ -121,6 +122,7 @@ public final class ConfigUtils {
                 .withConfigDataType(BlockRecordStreamConfig.class)
                 .withConfigDataType(BlockStreamJumpstartConfig.class)
                 .withConfigDataType(ModulesConfig.class)
+                .withConfigDataType(PathsConfig.class)
                 .withSource(new SimpleConfigSource().withValue("merkleDb.usePbj", false))
                 .withSource(new SimpleConfigSource().withValue("merkleDb.minNumberOfFilesInCompaction", 2))
                 .withSource(new SimpleConfigSource().withValue("merkleDb.maxFileChannelsPerFileReader", FILE_CHANNELS))

